### PR TITLE
Fix Missing more docs for some migrated .NET APIs

### PR DESCRIPTION
**Pull request by yishengjin1413**
Fix [Missing more docs for some migrated .NET APIs](https://mseng.visualstudio.com/CSI/_workitems?id=1076648&fullScreen=false&_a=edit)

### CODE REVIEW

File: add/metadata/System.Net/HttpWebResponse.meta.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ manager: "markl"
 ---
 
 ---
-uid: System.Net.HttpWebResponse.Dispose
+uid: System.Net.HttpWebResponse.Dispose*
```
Comment (mairaw) [inline add/metadata/System.Net/HttpWebResponse.meta.md]: this is different. the uids are usually for the specific APIs, not for the overload. should we undo these changes?

see example here: https://raw.githubusercontent.com/dotnet/docs/master/add/metadata/System.IO/StreamReader.meta.md

**Review by stevewhims — APPROVED** [review]
LGTM

File: add/metadata/System.Net/HttpWebResponse.meta.md (hunk 1 above)
Comment (yishengjin1413) [reply add/metadata/System.Net/HttpWebResponse.meta.md]: Hi Maira, 

Please check this page: [Dispose](https://www.microsoftonedoc.com/#/organizations/e6f6a65cf14f462597b64ac058dbe1d0/projects/3fedad16-eaf1-41a6-8f96-0c1949c68f32/containers/bcddac22-afb3-4f80-9ba0-ebbea45c68fa/tocpaths/a8530c58-2ee6-4e2c-972f-011154f8c87a/locales/en-US).

This item should be used for overload. We already have an item with UID: System.Net.HttpWebResponse.Dispose(System.Boolean) in this metadata file.

Comment (mairaw) [reply add/metadata/System.Net/HttpWebResponse.meta.md]: Sure. My point was just that I haven't seen the * being used in metadata files before.

**Review by mairaw — APPROVED** [review]
